build/Linux+BSD/portable/Recipe

```diff
@@ -92,12 +92,12 @@ export LD_LIBRARY_PATH="${PWD}/qt5/lib:$LD_LIBRARY_PATH"
 #wget http://pkgs.repoforge.org/rpmforge-release/rpmforge-release-0.5.3-1.el6.rf.$(arch).rpm
 #rpm -ivh rpmforge-release-0.5.3-1.el6.rf.$(arch).rpm || true # don't fail if already installed
 
-if [ "$(arch)" == "i686" ]; then
+if [[ "$(arch)" == "i686" ]]; then
```

Comment:
single [  ] shopuld work too, if using = rather than == and be more portable I believe